keycloak-config/pom.xml

```
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.alvearie</groupId>
+    <version>0.0.1-SNAPSHOT</version>
+    <artifactId>keycloak-config</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-admin-client</artifactId>
+            <version>11.0.2</version>
```

I think like to promote this to a top-level pom.xml `<dependencyManagement>` section.
However, since there's only a couple projects in the repo presently, that probably doesn't need to happen now.
Maybe just bump it to 12.0.2?
```suggestion
            <version>12.0.2</version>
```

---

actually, never mind, just let me handle that in a subsequent commit